docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,367 @@
+applicable, so it is suitable to be an example. Still, don't forget this is only 
+sample data, do not expect adequate results.
+
+We will create an {dfanalytics-job} to predict the minutes that flights are 
+delayed by. Each document in the dataset contains details for a single flight, 
+so this data is ready for analysis as it is already in a two-dimensional 
+entity-based data structure that we call a {dataframe}. Depending on your source 
+data, it may be required to first {ref}/transforms.html[transform] your source 
+data into an entity-centric index before you could setup and start the analysis.
+
+This is an example source document from the dataset:
+
+```
+{"_index":"kibana_sample_data_flights","_type":"_doc","_id":"S-JS1W0BJ7wufFIaPAHe","_version":1,"_seq_no":3356,"_primary_term":1,"found":true,"_source":{"FlightNum":"N32FE9T","DestCountry":"JP","OriginWeather":"Thunder & Lightning","OriginCityName":"Adelaide","AvgTicketPrice":499.08518599798685,"DistanceMiles":4802.864932998549,"FlightDelay":false,"DestWeather":"Sunny","Dest":"Chubu Centrair International Airport","FlightDelayType":"No Delay","OriginCountry":"AU","dayOfWeek":3,"DistanceKilometers":7729.461862731618,"timestamp":"2019-10-17T11:12:29","DestLocation":{"lat":"34.85839844","lon":"136.8049927"},"DestAirportID":"NGO","Carrier":"ES-Air","Cancelled":false,"FlightTimeMin":454.6742272195069,"Origin":"Adelaide International Airport","OriginLocation":{"lat":"-34.945","lon":"138.531006"},"DestRegion":"SE-BD","OriginAirportID":"ADL","OriginRegion":"SE-BD","DestCityName":"Tokoname","FlightTimeHour":7.577903786991782,"FlightDelayMin":0}}
+```
+
+
+{regression-cap} is a supervised machine learning analysis and therefore needs 
+to train on data that contains the ground truth for the `dependent_variable` 
+that we want to predict. In this example, the ground truth is available in each 
+document as the actual value of `FlightDelayMins`.
+
+If your source data consisted of documents that contains a `dependent_variable` 
+and some that does not contain a `dependent_variable`, then training is 
+performed on the `training_percent` of the documents that contain ground truth 
+and predictions will be made against all of the data. The current implementation 
+of {reganalysis} supports a single batch analysis for both training and 
+predictions. Future versions will allow a learned model to be incorporated at 
+index time.
+
+. Create a {dfanalytics-job}.
++
+--
+Use the {ref}/put-dfanalytics.html[create {dfanalytics-jobs}] API as you can see 
+in the following example:
+
+[source,console]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/model-flight-delays
+{
+  "source": {
+    "index": [
+      "kibana_sample_data_flights" <1>
+    ],
+    "query": { <2>
+      "range": {
+        "DistanceKilometers": { 
+          "gt": 0
+        }
+      }
+    }
+  },
+  "dest": {
+    "index": "df-flight-delays",  <3>
+    "results_field": "ml" 
```

Comment:
`results_field` is a default value. It can be removed from the Create statement (as it will show up in the response anyway. Slight simplification.